src/shared/isDevelopment.js

```diff
@@ -1,7 +0,0 @@
-let isDevelopment = !!(
```

Comment:
This function is intended to keep all logic related to environment selection in one place. Without this we're distributing this logic all over the code. If there is any concern about how environment is selected I'd reimplement this function.

Reply:
I tried to search other occurrences and didn't find any. Sorry, it just happened that after adding that `config.js` file having a separate `isDevelopment` file felt redundant.